cli/towerSysUtil.ml

```diff
@@ -103,5 +103,33 @@ module Dir = struct
       match err_opt with None -> make_exist ~perm path | Some err -> Some err
     ) None
 
+  let rec remove_dir path =
+    (* From http://pleac.sourceforge.net/pleac_ocaml/directories.html,
+ with some renaming for clarity *)
+    let rec process_tree f nodes =
+      Array.iter
+        (fun root ->
+         (match Unix.lstat root with
+          | {Unix.st_kind=Unix.S_DIR} ->
+             process_tree f
+                          (Array.map (Filename.concat root) (Sys.readdir root))
+          | _ -> ());
+         f root)
+        nodes in
+
+    let remove_node path =
+      match Unix.lstat path with
+      | {Unix.st_kind=Unix.S_DIR} ->
+         Printf.printf "rmdir %s\n%!" path;
+         Unix.rmdir path
+      | _ ->
+         Printf.printf "unlink %s\n%!" path;
```

Comment:
These should probably be calls to a logging module/function so the user can elect to see them with a (non-existent) command line flag like `-v`. This change can be made into an issue and done later.